src/pvsite_forecast.py

```diff
@@ -154,8 +154,16 @@ def convert_df(df: pd.DataFrame):
     csv = convert_df(df)
     now = datetime.now().isoformat()
 
+    #MAE Calculator
+    MAE = abs(df['generation_power_kw'] - df['forecast_power_kw']).sum() / len(df)
+    MAErounded = round(MAE/1000,ndigits=3)
+    if resample is None:
+         st.caption("Please resample to '15T' to get MAE")
+    else:
+         st.write("##### Current Mean Actual Error :", MAErounded, "MW")
```

Comment:
can you try using an f string
f"Mean Absolute Error {mae_rounded} MW}"

Reply:
Becasue we are using this page for UK sites too, where they are much smaller. Could you add an option, if mae_rounded_kw > 2000, then write MAE in MW, if not write MAW in kW

Reply:
so itll be something like
if  mae_rounded_kw > 2000:
   st.write(f'Mean Absolute Error {mae_rounded_kw} KW")
else:
  st.write(f'Mean Absolute Error {mae_rounded_mw} MW")